test/data/units.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I wanted to test a couple of specific features of reading units, so I crafted a simple model myself.

When we'll have the possibility to write models to disk we can delete this file and generate it dynamically (or maybe do everything in buffers, without passing through the disk)

Reply:
I guess this is perfectly okay now